src/DynamoCore/Models/DynamoModelCommands.cs

```diff
@@ -300,6 +310,36 @@ void BeginConnection(Guid nodeId, int portIndex, PortType portType)
             }
         }
 
+        void BeginMultipleConnections(Guid nodeId, int portIndex, PortType portType)
+        {
+            if (portType == PortType.Input) return; //only handle multiple connections when the port selected is an output port
+            activeStartPort = null;
+            var node = CurrentWorkspace.GetModelInternal(nodeId) as NodeModel;
+            if (node == null) return;
+
+            PortModel selectedPort = node.OutPorts[portIndex];
+
+            var connectorsForDeletion = new List<ModelBase>();
+            int numOfConnectors = selectedPort.Connectors.Count;
+            if (numOfConnectors == 0) return;
+
+            activeStartPort = selectedPort.Connectors[0].End;
```

Comment:
It's not used in that particular function, but I wasn't sure how much it will affect the program if it's left unassigned.
Perhaps we can have `activeStartPort` removed and just keep `activeStartPorts`?